src/components/layout/Layout.tsx

```diff
@@ -36,15 +46,14 @@ export default function Layout({
   figmaUrl,
   hideFooter = false,
   boxProps,
+  metaTitle,
```

Comment:
it looks like adding metaTitle forces you to use metaTitleInternal and this makes the code confusing. Could we name one of them headTitle to indicate the purpose better?

Reply:
@igoychev I'd rather rename it to `pageTitle` as `headTitle` and `metaTitle` are synonyms and it's also confusing.